cmake/modules/AddCephTest.cmake

```diff
@@ -1,5 +1,22 @@
 #AddCephTest is a module for adding tests to the "make check" target which runs CTest
 
+execute_process(
```

Comment:
if `ARCHSTR` is not used on `LINUX`, could you move this call `execute_process()` into the `elseif(FREEBSD)` block?